pkg/backends/calico/client.go

```diff
@@ -126,10 +126,34 @@ func NewCalicoClient(confdConfig *config.Config) (*client, error) {
 	c.watcherCond = sync.NewCond(&c.cacheLock)
 
 	// Increment the waitForSync wait group.  This blocks the GetValues call until the
-	// syncer has completed it's initial snapshot and is in sync.  The syncer is started
+	// syncer has completed its initial snapshot and is in sync.  The syncer is started
```

Comment:
Is this comment out of date?  We start the syncer in this method?

Reply:
Yes, outdated, I'll fix it.